kernel/kernel-api/src/main/java/io/delta/kernel/TransactionBuilder.java

```diff
@@ -113,6 +114,19 @@ TransactionBuilder withTransactionId(
    * @since 3.3.0
    */
   TransactionBuilder withTableProperties(Engine engine, Map<String, String> properties);
+  
+  /**
```

Comment:
should we document on case sensitivy of props: I see it differs for delta properties (case insensitive) vs arbitary properties (case sensitive)? Also for the withProperties method.

Reply:
Added a clarification